package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /etc/init.d/gluon-respondd reload
```

Comment:
It won't, trust me.

1. I tested this exact thing by creating scripts that all echo something and then `exec true`, they were all executed in order.
2. `/etc/hotplug.d/net/10-ar922x-led-fix` ends with `exit 0`

I have no idea what magic hotplug-call does with these scripts, as they don't need to be executable either, but they surely are not sourced.